Library/ViewModels/ManagePledgeViewModel.swift

```diff
@@ -107,9 +107,9 @@ public final class ManagePledgeViewModel:
       .takeWhen(cancelPledgeSelected)
       .filter { _, backing in backing.cancelable }
 
-    self.goToContactCreator = self.menuOptionSelectedSignal
-      .filter { $0 == .contactCreator }
-      .ignoreValues()
+    self.goToContactCreator = project
+      .takeWhen(self.menuOptionSelectedSignal.filter { $0 == .contactCreator })
+      .map { project in (MessageSubject.project(project), .backerModal)}
```

Comment:
What should the correct tracking context be for this screen?  Should I add a new context to the `MessageDialogContext`?